Tests/TestHelpers.swift

```diff
@@ -1,12 +1,12 @@
 import Moya
-import Foundation
 
-extension String {
-    var urlEscaped: String {
-        return self.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
-    }
-}
+#if os(OSX)
+import AppKit
+#else
+import Foundation
```

Comment:
`/Users/distiller/Moya/Tests/TestHelpers.swift:174:50: use of undeclared type 'CGFloat'; did you mean to use 'CGFloat'?` -- This is such a weird CI error.

I learned yesterday that CGFloat is not declared in Foundation. It's part of UIKit 🤔